source/_patterns/02-organisms/components/hero-banner.mustache

```diff
@@ -0,0 +1,54 @@
+<div class="hero-banner" role="region" data-behaviour="Hero-banner" {{#headingId}} aria-labelledby="{{headingId}}" {{/headingId}}>
```

Comment:
@discodavey would you expect the hero banner to be in molecules?

Reply:
My opinion is this would sit more in Organisms than Molecules. Happy to chat about it